great_expectations/rule_based_profiler/config/base.py

```diff
@@ -220,14 +220,18 @@ class RuleBasedProfilerConfig(BaseYamlConfig):
     def __init__(
         self,
         name: str,
+        class_name: str,
         config_version: float,
         rules: Dict[str, RuleConfig],
+        module_name: Optional[str] = None,
         variables: Optional[Dict[str, Any]] = None,
         commented_map: Optional[CommentedMap] = None,
         **kwargs
     ):
         self.name = name
+        self.class_name = class_name
         self.config_version = config_version
+        self.module_name = module_name
```

Comment:
@cdkini very minor: could we please keep class_name and module_name adjacent (in the same order as we have done for Checkpoint config)? Thanks!

Reply:
Please note comment above.